docker/muranoservice/README.md

```diff
@@ -0,0 +1,237 @@
+# Installing murano as service
+## Install murano packages
+
+Enable Openstack repositories:
+
+    apt install software-properties-common
+    add-apt-repository cloud-archive:newton
+
+Then, install murano packages:
+
+    apt-get install murano-api murano-engine python-muranoclient
+
+## Create database
+
+To create the database, complete these steps:
+
+Use the database access client to connect to the database server as the root user:
+
+    $ mysql -u root -p
+
+Create the murano database:
+
+    CREATE DATABASE murano;
+
+Grant proper access to the heat database:
+
+    GRANT ALL PRIVILEGES ON murano.* TO 'murano'@'localhost' \
+    IDENTIFIED BY 'MURANO_DBPASS';
+
+    GRANT ALL PRIVILEGES ON murano.* TO 'murano'@'%' \
+    IDENTIFIED BY 'MURANO_DBPASS';
+
+Replace MURANO_DBPASS with a suitable password.
+
+Exit the database access client.
+
+## Create endpoints
+
+Create the application-catalog service API endpoints:
+
+    $ keystone service-create --name application-catalog --type application-catalog \
+     --description "application-catalog"
+
+    $ keystone endpoint-create \
+     --service-id $(keystone service-list | awk '/ application-catalog / {print $2}') \
+     --publicurl http://controller:8082/v1/%\(tenant_id\)s \
+     --internalurl http://controller:8082/v1/%\(tenant_id\)s \
+     --adminurl http://controller:8082/v1/%\(tenant_id\)s \
+     --region regionOne
+
+## Configure murano
+Edit the /etc/murano/murano.conf file and complete the following actions:
+
+In the [database] section, configure database access:
+
+    [database]
+    ...
+    connection = mysql://murano:MURANO_DBPASS@controller/murano
+
+Replace MURANO_DBPASS with the password you chose for the Murano database.
+
+In the [DEFAULT] section, configure RabbitMQ message broker access:
+
+    [DEFAULT]
+    ...
+    rpc_backend = rabbit
+    rabbit_host = controller
+    rabbit_password = RABBIT_PASS
+
+Replace RABBIT_PASS with the password you chose for the guest account in RabbitMQ.
+
+In the [keystone_authtoken] and [ec2authtoken] sections, configure Identity service access:
+
+    [keystone_authtoken]
+    ...
+    auth_uri = http://controller:5000/v2.0
+    identity_uri = http://controller:35357
+    admin_tenant_name = service
+    admin_user = murano
+    admin_password = MURANO_PASS
+
+    [ec2authtoken]
+    ...
+    auth_uri = http://controller:5000/v2.0
+    Replace MURANO_PASS with the password you chose for the heat user in the Identity service.
+
+In the murano section, it is required to specified the default region in a multiregion environment
+
+    [murano]
+
+    region_name_for_services = RegionOne
+
+In the rabbitmq section [rabbitmq], which represents the rabbit among murano-engine and virtual machines
```

Comment:
among iff there are more than 2... :)

replace by between

add : at the end of the sentence.